crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -341,6 +341,8 @@ impl BaseClient {
         };
 
         // Find or create the room in the store
+        let is_a_new_room = store.room_exists(room_id).not();
```

Comment:
nit: here and in the parameters list, rename `is_new_room` maybe?

Also let's not abuse of the `not()` combinator and use a plain `!` please.

Reply:
Done.